extensions/amp-video-docking/0.1/test/test-amp-video-docking.js

```diff
@@ -142,16 +142,14 @@ describes.realWin('↗ 🔲', {amp: true}, env => {
   }
```

Comment:
Updated this test from non-standard internal generator `yield` to `async/await`.